### PR TITLE
Attack feedback + Dash controll changes

### PR DESCRIPTION
**Pull request by VBoschMatas**
What the title says. An AttackIndicator is required to see the feedback, its a .fbx called attack_indicator. 
**Important Note:** The model size does not scale with attack range, it has to be set manually (model's size is meant to be used with an attack of 3.5 range)
**Note 2:** To use the model use the .fbx called attack_indicator, BUT use its children AttackIndicator as the actual indicator linked to player's script.
Also some cleaning on PlayerController Serialization

### CODE REVIEW

File: Source/src/modules/ModuleSceneManager.cpp [hunk 1]
```diff
@@ -33,7 +33,7 @@ bool Hachiko::ModuleSceneManager::Init()
 #ifdef PLAY_BUILD
     main_scene->Start();
 #endif
-
+
```
Comment (VBoschMatas) [inline Source/src/modules/ModuleSceneManager.cpp:36]: Ehmmmm sure Github

**Comment by Bernatmago** [comment]
I think it could be a good idea to include a working scene to test this (If you already have one). It also helps to find importing issues that might be remaining

**Review by Bernatmago — APPROVED** [review]

**Comment by VBoschMatas** [comment]
> I think it could be a good idea to include a working scene to test this (If you already have one). It also helps to find importing issues that might be remaining.
> 
> We should even be able to store the player with screipts as a prefab (Not using the same name as the model or it would be overwriten on model reimport)

To be honest, I thought that I had a prefab of the character, gonna add that real quick. For the scene I haven't add one to keep development clean until we have a somewhat final scene of level 1, because right now its changing constantly. But if you think it is needed I can add one easily.

**Comment by VBoschMatas** [comment]
@Bernatmago There it is

**Comment by Bernatmago** [comment]
> > I think it could be a good idea to include a working scene to test this (If you already have one). It also helps to find importing issues that might be remaining.
> > We should even be able to store the player with screipts as a prefab (Not using the same name as the model or it would be overwriten on model reimport)
> 
> To be honest, I thought that I had a prefab of the character, gonna add that real quick. For the scene I haven't add one to keep development clean until we have a somewhat final scene of level 1, because right now its changing constantly. But if you think it is needed I can add one easily.

It was more in the sense of having it on the pr for easy testing but dont worry about it. Its true that we should not flood dev with them.

**Review by monikamnc — APPROVED** [review]
Super noiceeeee :3
Thanks <3

**Pull request merged**